Sources/PackageDescription/Resource.swift

```diff
@@ -51,39 +58,40 @@ public struct Resource: Encodable {
         self.path = path
         self.localization = localization
     }
-
-    /// Applies a platform-specific rule to the resource at the given path.
+    
+    /// Apply platform-specific rules to the resource at the given path.
```

Comment:
I think the convention is using 3rd-person present tense, so "Applies" instead of "Apply"

Reply:
Fixed, thank you!